### PR TITLE
London Class 8 - Ali Nurisiahsiahi - JavaScript- Week 1 

### PR DESCRIPTION
**Pull request by AliNurisiahsiahi**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Ali Nurisiahsiahi
- Your City: London
- Your Slack Name: Ali Nuri(Trainee)

# Homework Details

- Module: Javascript
- Week:1

# Notes

- What did you find easy? concatenation

- What did you find hard? decimal and function

- What do you still not understand?  work with git hub and git desktop 

- Any other notes? 

-----
[View rendered exercises/L-functions-nested/README.md](https://github.com/AliNurisiahsiahi/JavaScript-Core-1-Coursework-Week1/blob/master/exercises/L-functions-nested/README.md)

### CODE REVIEW

File: exercises/G-numbers/exercise.js [hunk 1]
```diff
@@ -1 +1,19 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let name = 'My nam is Ali. Number of students is :';
+let add = 'Number of mentors are :';
```
Comment (mairacagri) [inline exercises/G-numbers/exercise.js:3]: Hey Ali , great job on this assignment, by looking at your code i realize how can i improve mine, so thank you a lot!

Comment (AliNurisiahsiahi) [reply exercises/G-numbers/exercise.js:3]: Thank you for your feedback, it makes me stronger.

File: mandatory/4-tax.js [hunk 2]
```diff
@@ -17,7 +21,10 @@ function calculateSalesTax() {}
...
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+ afterAddTax = (price / 100) * 20
+ return `£${(afterAddTax + price).toFixed(2)}`
+}
 
 /* 
 ===================================================
```
Comment (dona-shehu) [inline mandatory/4-tax.js:30]: Well done Ali :+1:
You have done a great job :)

File: exercises/C-variables/exercise.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `greeting`
-
+var greeting = "Hello World"
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply exercises/C-variables/exercise.js:2]: I will do that.

File: exercises/D-strings/exercise.js [hunk 4]
```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
-
+let message =`This is string`
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply exercises/D-strings/exercise.js:2]: Yes, I am going to fix it.

File: exercises/E-strings-concatenation/exercise.js [hunk 5]
```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
-
-console.log(message);
+let greetingS ="Hi, may name is";
+let nameOf = "Ali";
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply exercises/E-strings-concatenation/exercise.js:3]: I will do that.

File: exercises/F-strings-methods/exercise.js [hunk 6]
```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply exercises/F-strings-methods/exercise.js:2]: Yes, it's helpful.

File: exercises/G-numbers/exercise.js [hunk 7]
```diff
@@ -1 +1,19 @@
...
+let total = 'They are totally :'
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply exercises/G-numbers/exercise.js:4]: Thank you for your advice.

File: exercises/J-functions/exercise.js [hunk 8]
```diff
@@ -1,7 +1,11 @@
 function halve(number) {
   // complete the function here
+  return number * 2;
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply exercises/J-functions/exercise.js:3]: Yes . Great.

File: mandatory/1-syntax-errors.js [hunk 9]
```diff
@@ -1,40 +1,37 @@
...
+function introduceMe(name, age) {
+  return "Hello, my name is " +name+ ' ' + "and I am " +age+ ' ' + "years old";
+}
 
 function getTotal(a, b) {
-  total = a ++ b;
-
-  return "The total is total";
+  total = a + b;
+  return "The total is" + ' ' + total;
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply mandatory/1-syntax-errors.js:12]: I will do that.

File: mandatory/3-function-output.js [hunk 10]
```diff
@@ -1,25 +1,36 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+// This function has a variable  getRandomNumber, within it is a return command, random number between 0 to .01 multiply by 10
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply mandatory/3-function-output.js:3]: Thanks, it's a good website.

File: exercises/G-numbers/exercise.js [hunk 11]
```diff
@@ -1 +1,19 @@
...
+
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+let sum = numberOfStudents + numberOfMentors;
+
+let count = `${name} ${numberOfStudents}`;
```
[the remark this answers is not in the document]
Comment (AliNurisiahsiahi) [reply exercises/G-numbers/exercise.js:10]: Thank you

**Pull request closed without merging**